Validation/RecoTrack/plugins/MultiTrackValidator.cc

```diff
@@ -453,13 +461,30 @@ size_t MultiTrackValidator::tpDR(const TrackingParticleRefVector& tPCeff,
         auto dR_tmp = reco::deltaR2(eta, phi, etaL[iTP2], phiL[iTP2]);
         if (dR_tmp<dR) dR=dR_tmp;
       }  // ttp2 (iTP)
+      if(cores != nullptr){
+        for (unsigned int ji = 0; ji < cores->size(); ji++) {//jet loop
+          if((*cores)[ji].pt() > ptMinJet_){
```

Comment:
This selection should be done once outside of the `tpDR` and `trackDR` functions (if not done outside MTV).

Reply:
ok, I'm working on it (still missing in the last commit)

Reply:
@makortel would TrackValidation_cff.py be the right place to define the external selector? is it ok to use CandPtrSelector or you have some other preference?

Reply:
@arizzi Yes, `TrackValidation_cff.py` is the right place. I think `CandPtrSelector` is fine (the more concrete `CaloJetRefSelector` would be a tiny bit better but it's unlikely to make a difference performance-wise, whereas more flexibility with `Cand` could be useful).